spec/controllers/api/v1/notes_controller_spec.rb

```diff
@@ -187,8 +187,9 @@
       end
 
       context 'when the note content length exceeds the limit for reviews' do
-        let(:user) { FactoryBot.create(:user) }
-        let(:utility) { user.utility }
+        let(:utility) { FactoryBot.create(%i[south_utility north_utility].sample) }
+        let(:user) { FactoryBot.create(:user, utility_id: utility.id) }
```

Comment:
Recordá que nos podemos ahorrar el `FactoryBot` dentro de los tests

```suggestion
        let(:utility) { create(%i[south_utility north_utility].sample) }
        let(:user) { create(:user, utility_id: utility.id) }
```